src/index.js

```diff
@@ -0,0 +1,147 @@
+// default values
+const state = {
+  temp: 79,
+  city: 'Seattle',
+};
+
+const getWeather = (lat, lon) => {
+  axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: {
+        lat: lat,
+        lon: lon,
+      },
+    })
+
+    .then((response) => {
+      // we updated the flask code to be in imperial units
+      state.temp = parseInt(response.data.current.temp);
+      const tempContainer = document.getElementById('temp');
+      tempContainer.textContent = state.temp;
+      changeTempColorAndGarden(state.temp);
+    })
+    .catch((error) => {
+      console.log("Couldn't get city temperature");
+    });
+};
+
+const getCityTemp = () => {
+  let city = document.getElementById('city').textContent;
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: city,
+      },
+    })
+    .then((response) => {
+      const latitude = parseFloat(response.data[0].lat);
+      const longitude = parseFloat(response.data[0].lon);
+      getWeather(latitude, longitude);
+    })
+
+    .catch((error) => {
+      console.log('Error in finding latitude and longitude.');
+    });
+};
+
+const getNewCity = () => {
+  const cityName = document.getElementById('city-name').value;
+  const cityContainer = document.getElementById('city');
+  cityContainer.textContent =
+    cityName[0].toUpperCase() + cityName.substring(1).toLowerCase();
+};
+
+const tempIncrease = () => {
+  state.temp += 1;
+  const tempContainer = document.getElementById('temp');
+  tempContainer.textContent = state.temp;
+  changeTempColorAndGarden(state.temp);
+};
+
+const tempDecrease = () => {
+  state.temp -= 1;
+  const tempContainer = document.getElementById('temp');
+  tempContainer.textContent = state.temp;
+  changeTempColorAndGarden(state.temp);
+};
+
+const changeTempColorAndGarden = (temperature) => {
+  const tempContainer = document.getElementById('temp');
+  const landscapeEmojis = document.getElementById('landscape');
+  if (temperature > 79) {
+    tempContainer.style.color = 'red';
```

Comment:
Style: Setting a class name (like `hot`) and then styling this with CSS would better follow [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns).